fhir-bulkimportexport-webapp/src/main/java/com/ibm/fhir/bulkimport/ImportPartitionCollector.java

```diff
@@ -80,10 +86,7 @@ public ImportPartitionCollector() throws Exception {
                 logger.finer("ImportPartitionCollector: Succeed get CosClient!");
             }
         }
-    }
 
-    @Override
-    public Serializable collectPartitionData() throws Exception{
```

Comment:
not sure why this change is needed, not inject property is used in ImportPartitionCollector(), the CosClient was created in ImportPartitionCollector() to make sure it's only created once for each ImportPartitionCollector object.

Reply:
it was null, always null when executed

Reply:
I debugged it, and found it was injected after object creation

Reply:
I am leaving this code as-is

Reply:
en... this is ugly. so this means the constructor is never called.

Reply:
The constructor is called... then injection... you're depending on the injection to happen prior to the constructor call .